src/common/attachments/attachmentsTable.component.tsx

```diff
@@ -220,6 +222,16 @@ function AttachmentsTable(props: AttachmentTableProps) {
     // Functions
     ...onPreservedStatesChange,
 
+    renderCreateRowDialogContent: ({ table }) => (
+      <UploadAttachmentsDialog
+        open={true}
+        onClose={() => {
+          table.setCreatingRow(null);
+        }}
+        entityId={entityId}
+      />
```

Comment:
```suggestion
<StyledUppyBox>
      <UploadAttachmentsDialog
        open={true}
        onClose={() => {
          table.setCreatingRow(null);
        }}
        entityId={entityId}
      />
 
</StyledUppyBox>
```

you need to wrap the dialog with this as this applies the styling to uppy. 

as in the uppy .utlis file change the zinde3x to use this 
```

  '& .uppy-Dashboard--modal .uppy-Dashboard-inner': {
    zIndex: theme.zIndex.modal,
  },
```